Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -776,11 +777,14 @@ PhysicalParticleContainer::AddPlasma (int lev, RealBox part_realbox)
         // The invalid ones are given negative ID and are deleted during the
         // next redistribute.
         const auto poffset = offset.data();
+        const bool rz_random_theta = m_rz_random_theta;
         amrex::ParallelForRNG(overlap_box,
         [=] AMREX_GPU_DEVICE (int i, int j, int k, amrex::RandomEngine const& engine) noexcept
         {
             IntVect iv = IntVect(AMREX_D_DECL(i, j, k));
             const auto index = overlap_box.index(iv);
+            Real theta_offset = 0._rt;
+            if (rz_random_theta) theta_offset = amrex::Random(engine) * 2._rt * MathConst::pi;
             for (int i_part = 0; i_part < pcounts[index]; ++i_part)
```

Comment:
Following Dave's comment:
```suggestion
#ifdef WARPX_DIM_RZ
            Real theta_offset = 0._rt;
            if (rz_random_theta) theta_offset = amrex::Random(engine) * 2._rt * MathConst::pi;
#endif
            for (int i_part = 0; i_part < pcounts[index]; ++i_part)
```